dotty-community-build/src/test/scala/dotty/communitybuild/CommunityProjectTest.scala

```diff
@@ -85,9 +85,6 @@ abstract class CommunityProjectTest(project: CommunityProject) {
       val exit = process.waitFor()
       assertEquals(exit, 0)
     }
-    exec("git", "clean", "-xfd")
```

Comment:
This will make it harder to reproduce tests, since the local clone can have dirty state.

Reply:
How can the submodule be in a dirty state?

Reply:
During local development, I wanted the tests to be a pure function from compiling the remove repos at a particular hash. As long as you don't `cd scopt && makeEdits()` then we have nothing to worry about.

Reply:
Is that OK if I keep `exec("git", "clean", "-xfd")` but still get rid of the two lines that follows?